install.sh

```diff
@@ -84,9 +85,6 @@ nvidia-modprobe -u -c0
 # reduces nvidia-smi invocation time 10x from 30 to 2 sec 
 # notable on large VM sizes with multiple GPUs
 # especially when nvidia-smi process is in CPU cgroup
```

Comment:
comments are needed ?

Reply:
Comments for context about why persistenced must be running, since we had an issue a while ago where persistenced not running => led to nvidia-smi being slow => which caused CSE to fail and => led to an outage for 8-GPU A100 users